server/container/guice/protocols/imap/src/main/java/org/apache/james/modules/protocols/IMAPServerModule.java

```diff
@@ -59,7 +58,6 @@ protected void configure() {
     }
 
     @Provides
-    @Singleton
```

Comment:
We discussed this with Quynh: It is done so that each IMAP server instance can get a different configuration (IE: configuration is done at the instance level for flexibility).

If we get a singleton, then each ImapProcessorChain will use the configuration of the last configured IMAP server instance.